src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/step/VariantLoaderStepParametersValidatorTest.java

```diff
@@ -0,0 +1,114 @@
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.parameters.validation.step;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.JobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+import uk.ac.ebi.eva.test.rules.PipelineTemporaryFolderRule;
+import uk.ac.ebi.eva.test.utils.TestFileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * Tests that the arguments necessary to run a {@link uk.ac.ebi.eva.pipeline.jobs.steps.VariantLoaderStep} are correctly
+ * validated
+ */
+public class VariantLoaderStepParametersValidatorTest {
+
+    private JobParametersValidator validator;
+
+    @Rule
+    public PipelineTemporaryFolderRule temporaryFolderRule = new PipelineTemporaryFolderRule();
+
+    private Map<String, JobParameter> parameters;
+
+    private Map<String, JobParameter> optionalParameters;
+
+    @Before
+    public void setUp() throws IOException {
+        validator = new VariantLoaderStepParametersValidator();
+        final File dir = TestFileUtils.getResource("/parameters-validation/");
+        final File inputVcf = TestFileUtils.getResource("/parameters-validation/file.vcf.gz");
+
+        dir.setReadable(true);
+        dir.setWritable(true);
+        inputVcf.setReadable(true);
+
+        parameters = new TreeMap<>();
+        parameters.put(JobParametersNames.DB_NAME, new JobParameter("database"));
+        parameters.put(JobParametersNames.DB_COLLECTIONS_VARIANTS_NAME, new JobParameter("variants"));
+        parameters.put(JobParametersNames.DB_COLLECTIONS_FILES_NAME, new JobParameter("files"));
+        parameters.put(JobParametersNames.INPUT_STUDY_ID, new JobParameter("inputStudyId"));
+        parameters.put(JobParametersNames.INPUT_VCF_ID, new JobParameter("inputVcfId"));
+        parameters.put(JobParametersNames.INPUT_VCF, new JobParameter(inputVcf.getCanonicalPath()));
+
+        optionalParameters = new TreeMap<>();
+        optionalParameters.put(JobParametersNames.INPUT_VCF_AGGREGATION, new JobParameter("NONE"));
+        optionalParameters.put(JobParametersNames.CONFIG_CHUNK_SIZE, new JobParameter("100"));
+    }
+
+    @Test
+    public void allJobParametersAreValid() throws JobParametersInvalidException, IOException {
+        validator.validate(new JobParameters(parameters));
+    }
+
+    @Test
+    public void allJobParametersIncludingOptionalAreValid() throws JobParametersInvalidException, IOException {
+        parameters.putAll(optionalParameters);
+        validator.validate(new JobParameters(parameters));
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void dbNameIsRequired() throws Exception {
```

Comment:
I'll go with is Required. That's what the test is checking. If the test fails it means that the field is not actually required.